plugin-server/src/cdp/consumers/cdp-person-performed-events.consumer.ts

```diff
@@ -0,0 +1,200 @@
+import { Message } from 'node-rdkafka'
+import { Counter, Gauge, Histogram } from 'prom-client'
+
+import { KAFKA_CDP_PERSON_PERFORMED_EVENT } from '../../config/kafka-topics'
+import { KafkaConsumer } from '../../kafka/consumer'
+import { runInstrumentedFunction } from '../../main/utils'
+import { CdpPersonPerformedEvent, Hub } from '../../types'
+import { parseJSON } from '../../utils/json-parse'
+import { logger } from '../../utils/logger'
+import { CdpConsumerBase } from './cdp-base.consumer'
+
+export const histogramPersonPerformedEventProcessing = new Histogram({
+    name: 'cdp_person_performed_event_processing_duration_ms',
+    help: 'Time spent processing person performed events',
+    labelNames: ['step'],
+    buckets: [1, 5, 10, 25, 50, 100, 250, 500, 1000, 2500],
+})
+
+export const cacheOperationsTotal = new Counter({
```

Comment:
Can we stop exporting counters :D we should only export things if they are needed elsewhere